e2e-tests/cypress/e2e/journeys/insurance/your-business/broker/validation/broker-are-you-using-a-broker-validation.spec.js

```diff
@@ -0,0 +1,87 @@
+import { broker } from '../../../../../pages/your-business';
+import partials from '../../../../../partials';
+import { submitButton } from '../../../../../pages/shared';
+import { ERROR_MESSAGES } from '../../../../../../../content-strings';
+import { ROUTES } from '../../../../../../../constants';
+import { EXPORTER_BUSINESS as FIELD_IDS } from '../../../../../../../constants/field-ids/insurance/exporter-business';
+
+const {
+  BROKER: {
+    USING_BROKER: FIELD_ID,
+  },
+} = FIELD_IDS;
+
+const {
+  ROOT,
+  START,
+  EXPORTER_BUSINESS: {
+    BROKER,
+    CHECK_YOUR_ANSWERS,
+  },
+} = ROUTES.INSURANCE;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.exporterBusiness;
+
+const BROKER_ERRORS = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS;
+const ERROR_MESSAGE = BROKER_ERRORS[FIELD_ID];
+
+const ERROR_ASSERTIONS = {
+  field: broker[FIELD_ID],
+  numberOfExpectedErrors: 1,
+  errorIndex: 0,
+};
+
+context('Insurance - Your business - Broker Page - As an Exporter I want to confirm that I am not using a broker for my export Insurance so that UKEF and I can easily collaborate and manage correspondence regarding my export insurance', () => {
```

Comment:
Can we rename this to have `-answer-no` prefix like in other tests? e.g `exporter-location-answer-no.spec.js`